h/static/styles/partials-v2/_annotation-card.scss

```diff
@@ -68,12 +68,22 @@
   display: flex;
   flex-direction: row;
   justify-content: flex-end;
+  align-items: baseline;
 }
 
-.annotation-card__incontext-link {
-  height: 10px;
-  width: 10px;
-  color: $grey-5;
+.annotation-card__footer .annotation-card__footer-link {
```

Comment:
Do we need the descendant selector here or can we just use the `.annotation-card__footer-link`?